catalyst_voices/lib/widgets/indicators/voices_no_internet_connection_banner.dart

```diff
@@ -0,0 +1,90 @@
+import 'package:catalyst_voices/widgets/widgets.dart';
+import 'package:catalyst_voices_assets/catalyst_voices_assets.dart';
+import 'package:catalyst_voices_localization/catalyst_voices_localization.dart';
+import 'package:flutter/material.dart';
+
+class NoInternetConnectionBanner extends StatelessWidget {
```

Comment:
Make sure to add a single sentence as documentation to explain what the widget does or how to use it.